corehq/apps/app_manager/tests/templatetags/test_xforms_extras.py

```diff
@@ -40,22 +35,6 @@ def test_secondary_language_includes_indicator(self):
         result = trans(translation_dict, langs=langs)
         self.assertEqual(result, 'English Output [en] ')
 
-    def test_do_not_use_delimiter_includes_html_indicator(self):
-        translation_dict = {'en': 'English Output'}
-        result = trans(translation_dict, use_delim=False)
-        self.assertEqual(result,
-            'English Output <span class="btn btn-xs btn-info btn-langcode-preprocessed">en</span> ')
-
-    def test_prefix_puts_indicator_in_front_of_translation(self):
-        translation_dict = {'en': 'English Output'}
-        result = trans(translation_dict, prefix=True)
-        self.assertEqual(result, ' [en] English Output')
-
-    def test_strip_tags_removes_tags_from_html_indicator(self):
-        translation_dict = {'en': 'English Output'}
-        result = trans(translation_dict, use_delim=False, strip_tags=True)
-        self.assertEqual(result, 'English Output en ')
-
```

Comment:
Why remove the tests? Are these combinations never used anywhere or something?

Reply:
Yeah. Initially I thought `trans` was a general-purpose method that could take any combination of parameters, but then I realized we're only ever using it to support the 4(?) filters that I extracted. Leaving in tests that didn't support real use cases (or were being tested through their respective filters) felt brittle.

Reply:
Okay yeah that makes total sense, thanks